duckduckgo/Android
# Update GPC copy

Task/Issue URL: https://app.asana.com/0/488551667048375/1199892618151767/f
Tech Design URL: 
CC: https://app.asana.com/0/1142021229838617/1200212320487140/f

**Description**:
We are updating GPC copy based on the conversations in https://app.asana.com/0/1142021229838617/1200212320487140/f


**Steps to test this PR**:
1. Go to settings
1. Click on Global Privacy Control
2. Ensure new copy is displayed
3. Learn more link is clickable and navigates to learn more site


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
